packages/components/src/components/modal/modal.tsx

```diff
@@ -146,7 +144,6 @@ const ModalElement = ({
             style={{
                 height: height || 'auto',
                 width: width || 'auto',
-                minHeight: min_height || 0,
```

Comment:
@arshad-rao-deriv I know @aizad-deriv used this property a couple of days ago in this file deriv-app/packages/cfd/src/Containers/cfd-top-up-demo-modal.tsx so I'm not sure it's a good idea to just delete the property which can be used somewhere. Please, recheck this with @aizad-deriv

Reply:
@sergei-deriv I found that min-height is already applied through css in the cfd-top-up-demo-modal.tsx so i don't think it's necessary. I have also checked in code base, we are not using it anywhere else.